website/content/en/docs/advanced-topics/multi-arch.md

```diff
@@ -4,13 +4,20 @@ linkTitle: Multiple Architectures
 weight: 80
 ---
 
-### Supporting Multiple Architectures
+# Supporting Multiple Architectures
 
-Authors may decide to distribute their bundles for various architectures: x86_64, aarch64, ppc64le, s390x, etc, to accommodate the diversity of Kubernetes clusters and reach a larger number of potential users. Each architecture requires however compatible binaries.
+Authors may decide to distribute their bundles for various architectures: x86_64, aarch64, ppc64le, s390x, etc, to accommodate the diversity of Kubernetes clusters and reach a larger number of potential users. Each supported architecture requires compatible binaries. Binary compatibility is based on the **platform**, which is generally comprised of an operating system, a CPU architecture, and other architecture variants. For this guide, we will treat the CPU architecture as the platform differentiator.
 
-#### Manifest lists
+That said, the same general advice would apply to supporting workloads on clusters of the same architecture but different operating systems (e.g. `linux/amd64`, `windows/amd64`). In this guide, we assume all workloads will be using the `linux` operating system.
 
-The most straightforward way of building operators and operands supporting multiple architectures is to leverage manifest lists, specified by [Image Manifest V2, Schema 2][manifest_list] or [OCI Image Index][image_index]. A manifest list points to specific image manifests for one or more platforms.
+## Fundamentals
+The basic principle of supporting multiple architectures is to ensure that each of your operator images is built for each of the architectures to be supported. From there, the images should be hosted in image registries as manifest lists. Finally, you'll need to update your distribution configuration to set which architectures are supported. This section explains each of these concepts in turn.
+
+### Building an Operator for Multiple Architectures
+[KubeBuilder explains][kubebuilder] how you can use `docker buildx` to build images for multiple architectures. Operator SDK leverages KubeBuilder to ensure that builds can be cross-architecture from the start.
```

Comment:
Nit: Kubebuilder shouldn't have a capital B for "builder" in it's name. Remove "explains" from the linking.
```suggestion
[Kubebuilder][kubebuilder] explains how you can use `docker buildx` to build images for multiple architectures. Operator SDK leverages Kubebuilder to ensure that builds can be cross-architecture from the start.
```

Reply:
Also curious about the term "cross-architecture", is that a standard? I'm assuming cross-platform is equivalent to cross-architecture but I've only ever heard the term cross-platform used.

Reply:
The note at the top of the doc explains why I use the word "architecture" as opposed to "platform" for this guide; however, I don't mind making the change for this specific call-out.

I am thread to thread the needle of balancing when we use platform (os+arch+variant) and when we use architecture. I've erred a little on the side of using architecture for this guide since it is a multi-architecture-specific guide and it's also the aspect that you'll build for most often when talking about workloads for enterprise computing.